golem_messages/validators.py

```diff
@@ -0,0 +1,12 @@
+from golem_messages import exceptions
+
+
+def validate_varchar(field_name, value, max_length):
+    if not (isinstance(value, str) and len(value) <= max_length):
```

Comment:
would it make sense to also allow a `max_length` of `None` and then only check for the instance type?

Reply:
I'd add it if it'd needed.